.github/CODEOWNERS

```diff
@@ -21,6 +21,7 @@
 /aws_pricing/                            @tsein-bc tsein@brightcove.com
 /backstage/                              @RoadieHQ support@roadie.io
 /bind9/                                  @ashuvyas45 ashuvyas45@gmail.com
+/blink/                                  @liavt liav@blinkops.com
```

Comment:
Could you also add this integration to [this section](https://github.com/DataDog/integrations-extras/blob/master/.github/CODEOWNERS#L166) so our doc team will be notified if needed? Thanks!

Reply:
I added it 👍

Reply:
🙇